x/wasm/keeper/proposal_integration_test.go

```diff
@@ -123,6 +123,80 @@ func TestInstantiateProposal(t *testing.T) {
 	require.NotEmpty(t, em.Events()[2].Attributes[0])
 }
 
+func TestInstantiateProposal_NoAdmin(t *testing.T) {
```

Comment:
Thanks for covering the additional cases! If you feel table tests are a better fit, then please feel free to refactor tests as well.